### ISSUE TITLE
Video Frame Focus Indicator not visible

### ISSUE THREAD

**Issue by mshubat** [ableplayer/ableplayer#526]
For Vimeo, it is possible to focus on the iframe of the player but there is no visual focus indicator present.

![image](https://user-images.githubusercontent.com/10766136/173374617-4cf37496-7214-47b8-8e6f-8de33b3f0084.png)

https://dequeuniversity.com/class/input-methods/keyboard-input/visual-focus-indicator

I believe the solution here may be to only allow focus on the overlay for vimeo (and add a visual focus indicator).

**Comment by terrill** [comment]
Closed with 6e3899d0f8841d0e4f19738dff2219dc8dc07935
